src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
`result` isn't informative 
```suggestion
  const newRobot = {};
```
```suggestion
  const inversedRobot = {};
```